x-pack/plugins/lists/common/schemas/request/patch_list_schema.ts

```diff
@@ -17,7 +17,15 @@ export const patchListSchema = t.intersection([
       id,
     })
   ),
-  t.exact(t.partial({ _version, description, meta, name })),
+  t.exact(
+    t.partial({
+      _version, // is undefined if not set during decode
```

Comment:
Is the `_version` the immutable/internal version for our use? Then user can set it's own version if it likes for `version`?

Reply:
The `_version` is for the optimistic concurrency control and to get conflicts from an earlier PR of here:
https://github.com/elastic/kibana/pull/72337

The `version` is for when we have immutable lists and want to version those to know which ones are loaded like rules are.